core/trino-main/src/main/java/io/trino/server/TrinoSystemRequirements.java

```diff
@@ -93,7 +93,7 @@ else if ("Mac OS X".equals(osName)) {
 
     private static void verifyJavaVersion()
     {
-        Version required = Version.parse("17.0.3");
+        Version required = Version.parse("17.0.4");
```

Comment:
Should we make this `17.0.9`? Is there a specific change between `17.0.3` and `17.0.4` that we require?

Reply:
Let's make it 17.0.5 as this is the version used in docker images and we've been testing with it for a while.

Reply:
17.0.5 is one year old. We're currently shipping 17.0.8, so that would mean we go backwards significantly.